tripal/src/Form/TripalImporterForm.php

```diff
@@ -119,6 +123,25 @@ public function buildForm(array $form, FormStateInterface $form_state, $plugin_i
       }
     }
 
+    // Adds a checkbox to indicate whether or not to publish the imported
+    // content. The default will be to publish (checked), uncheck to opt-out.
+    // An importer attribute can specify a list of bundles to publish,
+    // e.g. both 'gene' and 'mrna' for GFF importer.
+    if ($importer_def['publish']['bundle'] ?? FALSE) {
+      $bundles = (array) $importer_def['publish']['bundle'];
+      $form['publish'] = [
+        '#type' => 'fieldset',
+        '#title' => $this->t('Specify publishing behavior after importing'),
```

Comment:
We want to make sure it moves to just above the advanced fieldset.

```suggestion
        '#title' => $this->t('Specify publishing behavior after importing'),
        '#weight' => 8,
```

<img width="1499" height="848" alt="Screenshot 2025-09-23 at 10 51 40 AM" src="https://github.com/user-attachments/assets/5d534887-4c7f-4f1f-8f35-d8aa1b8ff3de" />